docs/deployment/connect-scm.md

```diff
@@ -33,38 +36,54 @@ The process to connect a source code manager depends on whether your SCM tool is
 
 If you opted to scan a GitHub or GitLab repository when you initially signed in, you may have already performed these steps and can skip to [Next steps](#next-steps).
 
+<Tabs
+    defaultValue="github-cloud"
+    values={[
+    {label: 'Azure DevOps Cloud', value: 'azure-devops-cloud'},
+    {label: 'Bitbucket Cloud', value: 'bitbucket-cloud'},
+    {label: 'GitHub Cloud', value: 'github-cloud'},
+    {label: 'GitLab Cloud', value: 'gitlab-cloud'},
+    ]}
+>
+
+<TabItem value='azure-devops-cloud'>
+
 ### Azure DevOps Cloud
```

Comment:
Since we have tabs now, I think these headers can be removed (this goes for all the H3s that are SCM names), unless you're keeping 'em to generate the right-hand nav bar?

Reply:
Yeah, I'll remove them. The h3 headers are nice but they seem "unclickable" when they're intabs (except for the tab currently displayed). Thanks for the catch.

Reply:
@khorne3 WDYT - here's a screenshot of both w/o headers and with:

![image](https://github.com/user-attachments/assets/3e63da43-9a33-4db6-970d-0b7dafeb18a9)

Reply:
I am very slightly in favor of removing the headers because the tabs are clear enough to me, but posting as a sanity check.

Reply:
I'm in favor of removing the headers; it looks cleaner and more deliberate to me